compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -988,8 +988,10 @@ bool OMR::X86::CodeGenerator::supportsAddressRematerialization()         { stati
 #undef CAN_REMATERIALIZE
 
 bool
-OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::DataType dt)
+OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::DataType dt, int32_t length)
```

Comment:
Shouldn't this accept ```VectorLength```

Reply:
Yes, I agree. It would be better to accept VectorLength. I will change.

Reply:
Pushed commit with this change.